scripts/scripts.js

```diff
@@ -642,6 +669,7 @@ async function loadLazy(doc) {
   sampleRUM('lazy');
   sampleRUM.observe(main.querySelectorAll('div[data-block-name]'));
   sampleRUM.observe(main.querySelectorAll('picture > img'));
+  centerArticleDivider(sectionDefaultArticles);
```

Comment:
One more thing here - this would run on every page. but you only want to run this script on article pages. so you should get the metadata of the page, check if the template is 'article' and then proceed to call your script. no point running this on other pages. This will help reduce the load on non article pages. Be as specific as possible when it comes to limiting running something in scripts.js that usually keeps things tight for performance.

Reply:
Hi @amol-anand , I added a template checker before running the scanTextNodes script. Please check once. Thank you!